inspirehep/config.py

```diff
@@ -69,6 +69,8 @@ def _(x):
 CELERY_ACCEPT_CONTENT = ['json', 'msgpack', 'yaml']
 CELERY_TIMEZONE = 'Europe/Amsterdam'
 CELERY_DISABLE_RATE_LIMITS = True
+CELERY_ALWAYS_EAGER = True
+CELERY_EAGER_PROPAGATES_EXCEPTIONS = True
```

Comment:
I assume these are year just for testing purposes. We should ship INSPIRE with eager mode on by default. Please remove it in the final PR.